ocaml/src/asd_server.ml

```diff
@@ -82,16 +82,20 @@ module DirectoryInfo = struct
 
   type t = {
     files_path : string;
-    directory_cache : (string, directory_status) Hashtbl.t
+    directory_cache : (string, directory_status) Hashtbl.t;
+    no_blobs : bool;
   }
 
-  let make files_path =
+  let make ?(no_blobs = false) files_path =
```

Comment:
maybe spare some brain cycles and replace the double negative with ?(write_blobs = true)